packages/server-wallet/src/protocols/close-channel.ts

```diff
@@ -46,7 +46,8 @@ function successfulWithdraw({app}: ProtocolState): boolean {
     .reduce((soFar, currentAi) => BN.add(soFar, currentAi.amount), BN.from(0));
   const amountTransferredToMe = app
     .funding(assetHolderAddress)
-    .transferredOut.filter(tf => tf.toAddress === myDestination)
+    // todo: figure out why destination in emitted events are lower case
+    .transferredOut.filter(tf => tf.toAddress === myDestination.toLocaleLowerCase())
```

Comment:
nit: I think you probably want `.toLowerCase()` (not sure the `locale` is doing anything for us, here.

Reply:
Good catch, I think `toLocaleLowerCase` was the first autocomplete option :) d8510e23e